src/app/clusters/ota-requestor/GenericOTARequestorDriver.cpp

```diff
@@ -351,28 +384,110 @@ void GenericOTARequestorDriver::DefaultProviderTimerHandler(System::Layer * syst
     SendQueryImage();
 }
 
-void GenericOTARequestorDriver::StartDefaultProviderTimer()
+void GenericOTARequestorDriver::StartPeriodicQueryTimer()
 {
     ChipLogProgress(SoftwareUpdate, "Starting the Default Provider timer, timeout: %u seconds",
```

Comment:
nit: Can you also update this log message?
```suggestion
    ChipLogProgress(SoftwareUpdate, "Starting the periodic query timer, timeout: %u seconds",
```